### PR TITLE
Add Mathieu Hofman (MHN)

### PR DESCRIPTION
**Pull request by robpalme**

### CODE REVIEW

**Comment by codehag** [comment]
@robpalme there is a Michal Hollman (MHN) with that abbreviation. Can you use a different one? For example MUH or MHO?

**Comment by codehag** [comment]
ping @robpalme

File: delegates.txt [hunk 1]
```diff
@@ -193,6 +193,7 @@ Mark Honenberg (MH)
 Mark S. Miller (MM)
 Masud Rahman (MRN)
 Mathias Bynens (MB)
+Mathieu Hofman (MHN)
```
Comment (pouwerkerk) [inline delegates.txt:196]: 👍

**Review by pouwerkerk — APPROVED** [review]
Thank you for adding Mathieu!

File: delegates.txt (hunk 1 above)
Comment (ljharb) [reply delegates.txt:196]: https://github.com/tc39/notes/pull/28#issuecomment-590362129

**Review by pouwerkerk — CHANGES_REQUESTED** [review]
Oh, I misunderstood the comment, I'll add a commit with an unused abbreviation.

**Comment by pouwerkerk** [comment]
I added a separate PR, namely #34, so @robpalme doesn't have to merge in my change before it could be added to this PR. Thanks again for doing the initial work, @robpalme!

**Comment by codehag** [comment]
closing in favor of #34

**Pull request closed without merging**